app/logic/utils.R

```diff
@@ -23,8 +30,29 @@ loading_screen <- function(text = "Loading...", bkg_color = "white") {
   )
 }
 
+# TODO finish the make_dropdown docs
+#' Title
+#' @param input_id
+#' @param text
+#' @param choices
+#' @returns
+#'
 #' @export
 make_dropdown <- function(input_id, text, choices) {
+  assert_string(input_id, min.chars = 1)
+  assert_string(text, min.chars = 1)
+  assert_vector(
+    x           = choices,
+    strict      = TRUE,
+    all.missing = FALSE,
+    min.len     = 1,
+    unique      = TRUE,
+    null.ok     = TRUE
```

Comment:
OK, I will get rid of it. I personally like it since it improves readability + I have an [addin](https://github.com/seasmith/AlignAssign) that maintains it (don't have to do it manually). But I see your point and will remove it